aw-server/src/main.rs

```diff
@@ -110,16 +117,26 @@ use std::path::PathBuf;
 // don't want this change?
 fn site_data_dir(app: Option<&str>, _: Option<&str>) -> Result<PathBuf, ()> {
     // Iterate over all XDG_DATA_DIRS and return first match that exists
-    if let Some(joined) = env::var_os("XDG_DATA_DIRS") {
-        for mut data_dir in env::split_paths(&joined) {
-            if let Some(app) = app {
-                data_dir.push(app);
+    let joined = match env::var_os("XDG_DATA_DIRS") {
+        // If $XDG_DATA_DIRS is either not set or empty, a value equal to /usr/local/share/:/usr/share/ should be used.
+        Some(path) => {
+            if path.is_empty() {
```

Comment:
Doesn't look like this allows several values in `XDG_DATA_DIRS`? No biggie, but technically not up to spec so might break if users have `XDG_DATA_DIRS` set to several paths.

Should also probably log a warning message if the path set by the env var is being ignored.

Reply:
I'm sorry, I didn't understand what you meant. [Here](https://github.com/ActivityWatch/aw-server-rust/pull/262/files/fb2d067ccf3255429a55c145c764975b8c5eb2d4#diff-39252f270b66dc7653d440ab41e5f0050cd2f1abaf3387dafa033be75cf22b24R132), we traverse all paths contained in XDG_DATA_DIRS.
I tested it on my computer and the program works correctly when XDG_DATA_DIRS contains multiple paths.

Reply:
Ah, I was confused about what happened there. I thought `is_empty()` meant if the *directory* was empty. All good then!